src/services/measure/measure.js

```diff
@@ -0,0 +1,270 @@
+goog.provide('ngeo.interaction');
+goog.provide('ngeo.interaction.Measure');
+
+goog.require('goog.events');
+goog.require('ol.DrawEvent');
+goog.require('ol.Feature');
+goog.require('ol.FeatureOverlay');
+goog.require('ol.MapBrowserEvent');
+goog.require('ol.Observable');
+goog.require('ol.Overlay');
+goog.require('ol.interaction.Pointer');
+goog.require('ol.style.Style');
+
+
+
+/**
+ * @classdesc
+ * Interaction that allows measuring (length, area, ...).
+ *
+ * @constructor
+ * @extends {ol.interaction.Pointer}
+ */
+ngeo.interaction.Measure = function() {
+
+
+  /**
+   * The help tooltip element.
+   * @type {Element}
+   * @private
+   */
+  this.helpTooltipElement_;
+
+
+  /**
+   * Overlay to show the help messages.
+   * @type {ol.Overlay}
+   * @private
+   */
+  this.helpTooltip_;
+
+
+  /**
+   * The measure tooltip element.
+   * @type {Element}
+   * @private
+   */
+  this.measureTooltipElement_;
+
+
+  /**
+   * Overlay to show the measurement.
+   * @type {ol.Overlay}
+   * @private
+   */
+  this.measureTooltip_;
+
+
+  /**
+   * Pointer move listener key
+   * @type {goog.events.Key}
+   * @private
+   */
+  this.pointerMoveHandlerKey_;
```

Comment:
As you are inheriting from `ol.interaction.Pointer`, there is already a method `handleMoveEvent` that you could overwrite.